source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -228,17 +232,19 @@ Install the Concordium Ledger app on MacOS
 
       ./install.sh
 
-2. The Ledger says **Deny unsafe manager**. Press the right button to navigate through the public key until the Ledger says **Allow unsafe manager**. Press both buttons. The Ledger says **Loading, please wait** while it installs the app.
+#. The Ledger says **Deny unsafe manager**. Press the right button to navigate through the public key until the Ledger says **Allow unsafe manager**. Press both buttons. The Ledger says **Loading, please wait** while it installs the app.
 
-3. The Ledger says **Concordium**. Press both buttons. The Ledger says **Concordium is ready**. You can now use the Ledger with the Desktop Wallet.
+#. The Ledger says **Concordium**. Press both buttons. The Ledger says **Concordium is ready**. You can now use the Ledger with the Desktop Wallet.
 
 .. _update-app-macos:
 
-Update the Concordium Ledger app on macOS
--------------------------------------------
+Update/reinstall the Concordium Ledger app on macOS
+---------------------------------------------------
 
 For the app to work properly with the current version of the Desktop Wallet, make sure that you update to the latest version of the app.
 
+When you update your Ledger, it removes the apps but not the certificates. So you do not need to do anything about the certificate.
```

Comment:
I am not sure that apps _always_ are deleted when you update your Ledger.
Either write that apps _may_ be deleted, or investigate to figure out whether it always happens.